rmgpy/tools/simulate.py

```diff
@@ -93,12 +93,11 @@ def simulate(rmg, diffusionLimited=True):
         
         else:
             logging.info('Conducting simulation of reaction system %s...' % (index+1))
-            
-        if rmg.saveSimulationProfiles:
-            reactionSystem.attach(SimulationProfileWriter(
-                rmg.outputDirectory, index, rmg.reactionModel.core.species))   
-            reactionSystem.attach(SimulationProfilePlotter(
-                    rmg.outputDirectory, index, rmg.reactionModel.core.species))
+
+        reactionSystem.attach(SimulationProfileWriter(
+            rmg.outputDirectory, index, rmg.reactionModel.core.species))
+        reactionSystem.attach(SimulationProfilePlotter(
+            rmg.outputDirectory, index, rmg.reactionModel.core.species))
```

Comment:
What was the reason for removing the `if` statement?

Reply:
Sorry, I just read your commit message. What if my purpose is running sensitivity and I don't want simulation profiles?

Reply:
Sure, that's possible, but it doesn't really take any extra time to write the profiles and I figure it's better to have it by default rather than having to rerun a very long simulation just to get the profiles if one forgot to turn on `saveSimulationProfiles`.

Reply:
Ok, that's fine.